internal/controller/controllers/bmo_utils.go

```diff
@@ -91,6 +106,37 @@ func (r *bmoUtils) GetIronicIPs() ([]string, []string, error) {
 	return ironicIPs, inspectorIPs, nil
 }
 
+func (r *bmoUtils) GetICCConfig(ctx context.Context) (*ICCConfig, error) {
```

Comment:
This doesn't need to be exported if we're only using it from this package.

Reply:
I changed it, not sure if having a private method in BMOUtils interface is very idiomatic, but it looks like it's mainly here to get a mock out of it. Let me know if we should do differently :thinking: